tests/functional/File_Logging_Test.php

```diff
@@ -0,0 +1,174 @@
+<?php
+namespace Tribe\Common;
+
+use Codeception\TestCase\WPTestCase,
+    Tribe__Log,
+    Tribe__Log__Logger,
+    Tribe__Main;
+
+class Logging_Test extends WPTestCase {
```

Comment:
This, along with the file based logger, should use WP Filesystem API and mock it.
Either "manually" or using something like [a mock of it](https://github.com/JDGrimes/wp-filesystem-mock).